ws-nextjs-app/pages/[service]/new_live/[id]/Post/index.tsx

```diff
@@ -59,40 +59,26 @@ const PostContent = ({ contentBlocks }: { contentBlocks: OptimoBlock[] }) => {
   );
 };
 
-const PostItem = ({ postItem }: { postItem: Post }) => {
+const Post = ({ post }: { post: PostType }) => {
   const headerBlocks = pathOr<OptimoBlock[]>(
     [],
     ['header', 'model', 'blocks'],
-    postItem,
+    post,
   );
 
   const contentBlocks = pathOr<OptimoBlock[]>(
     [],
     ['content', 'model', 'blocks'],
-    postItem,
+    post,
   );
 
   return (
-    <div>
+    <>
       <PostHeadings headerBlocks={headerBlocks} />
       <PostContent contentBlocks={contentBlocks} />
       <hr />
```

Comment:
Not part of this PR I realise but just spotted this, can we move this hr element please?

Reply:
We should probably do this in the Post ux + a11y work, just to keep things coupled https://jira.dev.bbc.co.uk/browse/WSTEAM1-448. I've added it as a point on the ticket